draft-ietf-quic-reliable-stream-reset.md

```diff
@@ -146,8 +146,11 @@ up to that byte offset are lost, the initiator MUST retransmit this data, as
 described in ({{Section 13.3 of RFC9000}}). Data sent beyond that byte offset
 SHOULD NOT be retransmitted.
 
-As described in {{Section 3.2 of RFC9000}}, it MAY deliver data beyond that
-offset to the application.
+As described in {{Section 3.2 of RFC9000}}, QUIC stacks MAY deliver data beyond
+that offset to the receiving application. An sending QUIC stack MAY choose a
+Reliable Size that is greater than the offset supplied by the application.
+Applications MUST NOT assume that the error code or the offset traverses to their
+peers.
```

Comment:
See https://github.com/quicwg/reliable-stream-reset/issues/33#issuecomment-1765653993, this is part of the API contract, and I don't think we should specify anything here.

Reply:
I disagree, let's continue discussing this on #33.